plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/model/OptOutPlugins.java

```diff
@@ -0,0 +1,24 @@
+package io.jenkins.tools.pluginmodernizer.core.model;
+
+import java.util.Arrays;
+
+public enum OptOutPlugins {
+    // List of plugins that have opted out from the modernization by plugin-modernizer-tool
+    SAFE_BATCH_ENVIRONMENT_FILTER("safe-batch-environment-filter"),
```

Comment:
I think it would be better to have a file that is not dependent of the release of plugin modernizer